src/routers/UserRouter.ts

```diff
@@ -104,7 +104,10 @@ const getProfileRouter = (basePath: string = '/', routePath: string = '/users/pr
         given_name: firstName && firstName.trim() ? firstName.trim() : user?.given_name,
         family_name: lastName && lastName.trim() ? lastName.trim() : user?.family_name,
       };
-      const userUpdated = await authMgmtService.updateUser(userId, update);
+      const userUpdated = await authMgmtService.updateUser(userId, {
```

Comment:
Can we remove the above validations? (trim() on the inputs) they are not necessary anymore since we use express-validator.

```
const firstName = get(req.body, 'firstName', user?.given_name)
const lastName = get(req.body, 'lastName', user?.family_name)

const update = {
  given_name: firstName,
  family_name: lastName,
  name: `${firstName} ${lastName}`,
};
```